api/opencl_runtime_layer.asciidoc

```diff
@@ -14690,6 +14691,32 @@ If using layered extension {cl_khr_command_buffer_mutable_dispatch_EXT},
 usage>>.
 ====
 
+Simultaneous use is defined using the _prerequisite_ terminology from the
+<<_execution_model, execution model>>.
+ifndef::cl_khr_command_buffer_multi_device[]
+A command-buffer exhibits undefined behavior if a simultaneous use usage
```

Comment:
The expression "use usage" seems redundant. I believe the word "usage" can be omitted.